src/components/view_file.vue

```diff
@@ -23,14 +25,24 @@
     incoming_filename!: string;
 
     @Prop({required: true, type: String})
-    incoming_file_content!: string;
+    incoming_file_contents!: string;
+
+    @Prop({default: () => {}, type: Object})
+    incoming_height_specifications!: object;
```

Comment:
Does this need to be an object, or can it just be a string and then put :style="{height: <height>}" in the template?